oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -881,9 +993,17 @@ int encodeEventMsg(uint8_t eventType, const std::vector<uint8_t>& eventDataVec,
     return rc;
 }
 void pldm::responder::oem_ibm_platform::Handler::setHostEffecterState(
-    bool status)
+    bool status, uint16_t entityTypeReceived)
 {
-    pldm::pdr::EntityType entityType = PLDM_ENTITY_PROC;
+    pldm::pdr::EntityType entityType;
+    if (entityTypeReceived == PLDM_ENTITY_MEMORY_MODULE)
+    {
+        entityType = PLDM_ENTITY_MEMORY_MODULE;
+    }
+    else
```

Comment:
Would be good to check like "else if (entityTypeReceived == PLDM_ENTITY_PROC)". Also add "else" and return.

Reply:
Done